packages/design/src/components/badge/_badge.scss

```
@@ -1,14 +1,15 @@
 @use "../../core/utils/functions" as *;
+@use "_variables" as *;
 
 $BADGE_SELECTOR: ".badge" !default;
 
-@mixin make-badge-variant($BADGE_SELECTOR, $background-color, $border-color: "") {
-    background-color: $background-color;
+@mixin make-badge-variant($BADGE_SELECTOR, $color-background, $color-border: "") {
+    color-background: $color-background;
 
-    @if $border-color != "" {
-        border-color: $border-color;
+    @if $color-border != "" {
+        color-border: $color-border;
     } @else {
-        border-color: $background-color;
+        color-border: $color-background;
```

Såg inte detta när jag kollade tidigare men här måste det stå kvar `border-color` på vänster sida. Gäller `background-color` också.

```suggestion
        border-color: $color-background;
```

Anledningen är för att det är så de heter i CSS:

* [background-color](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color)
* [border-color](https://developer.mozilla.org/en-US/docs/Web/CSS/border-color)

Det vi har på högersidan är våra namn, där kan vi använda "color-background".